### ISSUE TITLE
Auto updates on MacOS doesn't work

### ISSUE THREAD

**Issue by Moskovych** [lensapp/lens#3367]
**Describe the bug**
Lens installed via `brew` from versions 4.x or 5.0.0.
During the launch it show the info there is a new version available (see screenshot section)

If click `yes` - nothing is happening.

**To Reproduce**
1. Install Lens via `brew` version 5.0.0
2. Open Lens
3. In the popup for available version click `Yes` - to update
4. See error (if debug is enabled)

**Expected behavior**
2 possible ways:
1. Remove `auto_updates` from brew and do upgrades via `brew upgrade` command
2. Do auto updates smoothly. Show the error if such happens without diving into the debug mode.

**Screenshots**
<img width="523" alt="Screenshot 2021-07-13 at 10 57 22 AM" src="https://user-images.githubusercontent.com/10400095/125414037-74184344-6b7c-4bf5-832b-5219b7e3a191.png">

**Environment (please complete the following information):**
- Lens Version: 5.0.0-latest.20210629.3
- OS: MacOS Big Sur 11.4
- Installation method (e.g. snap or AppImage in Linux): `brew`

**Logs:**
Logs with enabled debug mode:
```
info: Update has already been downloaded to /Users/moskovych/Library/Application Support/Caches/lens-updater/pending/Lens-5.0.2-latest.20210705.2-mac.zip).
info: [UPDATE-CHECKER]: broadcasting update available {"backchannel":"auto-update:5.0.2-latest.20210705.2","version":"5.0.2-latest.20210705.2"}
info: [UPDATE-CHECKER]: User chose to update now
info: / requested
info: /1626162735450-6498.zip requested
info: /1626162735450-6498.zip requested by Squirrel.Mac, pipe /Users/moskovych/Library/Application Support/Caches/lens-updater/pending/Lens-5.0.2-latest.20210705.2-mac.zip
2021-07-13 10:52:19.910 Lens[46806:618625] Download completed to: file:///Users/moskovych/Library/Caches/com.electron.lens.ShipIt/update.2aE8eIh/1626162735450-6498.zip
warn: Could not locate update bundle for com.electron.lens within file:///Users/moskovych/Library/Caches/com.electron.lens.ShipIt/update.2aE8eIh/ {"code":2,"domain":"SQRLUpdaterErrorDomain"}
error: Error: Error: Could not locate update bundle for com.electron.lens within file:///Users/moskovych/Library/Caches/com.electron.lens.ShipIt/update.2aE8eIh/
```

**Kubeconfig:**
N/A

**Additional context**
N/A

**Comment by tomislater** [comment]
On windows also.

**Comment by Nokel81** [comment]
That is a known issue that cannot be resolved except by manually installing the latest version. Sorry about the bother.

**Comment by tomharrisonjr** [comment]
I have a version downloaded from GitHub and it also doesn't update automatically

**Comment by Nokel81** [comment]
@tomharrisonjr You must be running 4.2.5 then?

**Comment by Moskovych** [comment]
@Nokel81 , if it's known issue, wouldn't it be the same issue with next updates?

**Comment by Nokel81** [comment]
No we fixed it in 5.0.2
